src/files-and-videos/videos-page/VideosPage.test.jsx

```diff
@@ -421,7 +421,7 @@ describe('Videos page', () => {
               fireEvent.click(screen.getByText(messages.applySortButton.defaultMessage));
             });
 
-            const imageFilterChip = screen.getByTestId('icon-after');
+            const imageFilterChip = screen.getByRole('button', { name: 'Remove this filter' });
```

Comment:
The `testid` was removed from the "after" button in upstream Paragon. And anyways `testid` should be avoided. So I added an explicit alt text using the new `iconAfterAlt` attribute, and updated the test to use that.